src/commands/config.rs

```diff
@@ -17,18 +17,37 @@ pub fn set_file_mode(file: &PathBuf) {
         .expect("could not set permissions on file");
 }
 
-pub fn global_config(
-    email: String,
-    api_key: Option<String>,
-    api_token: Option<String>,
-) -> Result<(), failure::Error> {
-    let s = GlobalUser {
-        email,
-        api_key,
-        api_token,
+pub fn global_config(token: bool) -> Result<(), failure::Error> {
+    let mut user = GlobalUser {
+        email: None,
+        api_key: None,
+        api_token: None,
     };
 
-    let toml = toml::to_string(&s)?;
+    if token {
+        println!("Enter API token: ");
+        let mut api_token_str: String = read!("{}\n");
+        api_token_str.truncate(api_token_str.trim_end().len());
+        if !api_token_str.is_empty() {
+            user.api_token = Some(api_token_str);
+        }
+    } else {
+        println!("Enter email: ");
+        let mut email_str: String = read!("{}\n");
+        email_str.truncate(email_str.trim_end().len());
+        if !email_str.is_empty() {
+            user.email = Some(email_str);
+        }
+
+        println!("Enter API key: ");
```

Comment:
nit: can we make this `Enter global API key: ` to differentiate them a little bit?

Reply:
possible idea: maybe print a warning saying you can use API tokens instead of global API keys? not sure if we need it or want it since it's in beta but perhaps would be useful?

Reply:
sure!